Makefile.win

```diff
@@ -1,4 +1,4 @@
-# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+﻿# Licensed under the Apache License, Version 2.0 (the "License"); you may not
```

Comment:
What is this change?

Reply:
Looks like you inserted a Unicode character at the start of this file. Please revert; it breaks `Makefile.win`.